core/tests/test_populate_db.py

```diff
@@ -20,26 +20,24 @@
     CONST_SENSOR_LOCATION_TESTS
 )
 
-from crop.db import create_database, connect_db, drop_db
+from core.crop.db import create_database, connect_db, drop_db
 
-from crop.ingress_adv import advanticsys_import
+from core.crop.ingress_adv import advanticsys_import
 
-from crop.populate_db import session_open, session_close, insert_advanticsys_data
+from core.crop.populate_db import session_open, session_close, insert_advanticsys_data
```

Comment:
Instead of adding core., could you please adjust your python path since we have restructured the repository.